module2/src/com/epam/sorting/SortingImplementer.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.sorting;
+
+public class SortingImplementer {
+
+    //task 1
+    public static int[] InsertArrayIntoArray(int[] longArray, int[] shortArray, int insertIndexIntoArray) {
```

Comment:
```suggestion
    public static int[] insertArrayIntoArray(int[] longArray, int[] shortArray, int insertIndexIntoArray) {
```

In addition you can use not only long and short arrays, so please rename the variables